config/eslint.config.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = {
+    parser: 'babel-eslint',
+    env: {
+        browser: true,
+        commonjs: true,
+        es6: true,
+        jest: true,
+        node: true
+    },
+    plugins: [
+        'jsx-a11y',
```

Comment:
I just stumbled into this as well, hadn't used it yet. 🤔 👍

Reply:
If anyone else is wondering, `a11y` stands for `accessibility`. The number in the middle is how many letters are in there. In the same vein `i18n` stands for `internationalization`